### PR TITLE
Nano: add Unet ut for PyTorch openvino & ipex+jit

### PR DESCRIPTION
**Pull request by rnwang04**
## Description

Add UNet related UT in PR action to prevent changes on Nano side from causing stable diffusion to fail.

### 1. Why the change?
https://github.com/intel-analytics/BigDL/pull/7500

### 2. User API changes
None, just add two unittests.

### 3. Summary of the change

- add twos UNet related UT
- extend Tensor conditional_args from `torch.Tensor` to `(torch.Tensor, torch.FloatTensor, torch.LongTensor)` to cope more Tensor types

### 4. How to test?
- [x] Unit test

### CODE REVIEW

File: python/nano/test/openvino/pytorch/test_openvino.py [hunk 1]
```diff
@@ -303,3 +303,33 @@ def test_openvino_trace_kwargs(self):
...
+        unet(image_latents, torch.Tensor([980]).long(), encoder_hidden_states)
+        dynamic_axes= {"sample": [0],
+                        "encoder_hidden_states": [0],
+                        "unet_output": [0]}
+        nano_unet = InferenceOptimizer.trace(unet, accelerator="openvino",
+                                                input_sample=input_sample,
+                                                input_names=["sample", "timestep",
+                                                            "encoder_hidden_states", "return_dict"],
+                                                output_names=["unet_output"],
+                                                dynamic_axes=dynamic_axes,
```
Comment (TheaperDeng) [inline python/nano/test/openvino/pytorch/test_openvino.py:329]: I wonder what will happen if we leave this `dynamic_axes` to be autogenerated

**Review by TheaperDeng — COMMENTED** [review]
others LGTM

File: python/nano/test/openvino/pytorch/test_openvino.py (hunk 1 above)
Comment (rnwang04) [reply python/nano/test/openvino/pytorch/test_openvino.py:329]: Just validate it, no need for `dynamic_axes`, `input_names` and `output_names` now. Will update the ut. @cyita

Comment (rnwang04) [reply python/nano/test/openvino/pytorch/test_openvino.py]: If leave this `dynamic_axes` to be autogenerated, then model can't deal with dynamic shape which is showed in uts.

**Review by cyita — APPROVED** [review]

**Comment by cyita** [comment]
LGTM

**Pull request merged**